queue.go

```diff
@@ -107,6 +110,10 @@ func (opt *QueueOptions) Init() {
 		opt.WaitTimeout = 10 * time.Second
 	}
 
+	if opt.ZombieConsumerTimeout == 0 {
+		opt.ZombieConsumerTimeout = 6 * time.Hour
+	}
```

Comment:
What about `ConsumerIdleTimeout`? Looks cleaner to me.

Reply:
done